addons/web/views/webclient_templates.xml

```diff
@@ -100,7 +100,7 @@
             <link rel="stylesheet" href="/web/static/src/css/data_export.css"/>
             <link rel="stylesheet" href="/base/static/src/css/modules.css"/>
 
-            <link rel="stylesheet" href='/website/static/src/less/import_bootstrap.less'/>
+            <link rel="stylesheet" href='/web/static/src/less/import_bootstrap.less'/>
```

Comment:
Have tryed, no idea, why every push to my branch is added to the PR. But as I can say yelizariev, there is more broken, in class Field, which is a Datadescriptor. Iam still in analysing, but as I can say until now, it seems, that the __getter__, __setter__ methods are not correctly, cause there is already __get__, __set__, __delete__ defined. So that recursion depth exceeded seems to be typical error coder does in a Datadescriptor. Watching https://www.youtube.com/watch?v=v2WTVCyTYMw  start at 50:10 .